tests/wpt/metadata-css/css-conditional-3_dev/html/001.htm.ini

```diff
@@ -0,0 +1,8 @@
+[001.htm]
+  type: testharness
+  [Inserting @font-face inside @supports works]
+    expected: FAIL
+
+  [Inserting an @supports inside a style rule should fail]
```

Comment:
This test fails because the previous test, which attempts to insert a rule, fails to do so, messing up the indices.